### ISSUE TITLE
Custom timestamping servers

### ISSUE THREAD

**Issue by laurentsimon** [sigstore/model-transparency#205]
This would require a URL (+ optionally an expected identity?) to be configured

**Comment by mihaimaruseac** [comment]
Should this be handled by the signing infra instead of the library?

**Comment by haydentherapper** [comment]
+1, this is on the roadmap for sigstore-python

FYI @woodruffw

**Comment by di** [comment]
https://github.com/sigstore/sigstore-python/issues/349 is the relevant issue.

**Comment by woodruffw** [comment]
Yep! We have that queued up for our current phase of work, so we'll have an update on signing API side support shortly 🙂
